packages/wasm/README.md

```diff
@@ -53,6 +53,22 @@ Default: `null`
 
 Specifies an array of strings that each represent a WebAssembly file to load synchronously. See [Synchronous Modules](#synchronous-modules) for a functional example.
 
+### `limit`
```

Comment:
could we go with `maxFileSize` here instead?

Reply:
Sure - I chose `limit` because it is comparable to the `limit` option of the URL plugin.